### PR TITLE
add option useColorPicker

### PR DESCRIPTION
**Pull request by gilliM**

### CODE REVIEW

**Comment by sbrunner** [comment]
Thanks, merge :-)

**Comment by sbrunner** [comment]
If tests pass :-)

**Comment by asaunier** [comment]
> Thanks, merge :-)

This will be a merge to branch colors_for_search (not master or 1.5) created by @ger-benjamin...
@ger-benjamin What do you think?

File: core/src/script/CGXP/plugins/FullTextSearch.js [hunk 1]
```diff
@@ -178,9 +186,11 @@ cgxp.plugins.FullTextSearch = Ext.extend(gxp.plugins.Tool, {
             map: map
         }, this.widgetOptions));
 
-        colorpickerWidth = 35
-        combo.items.insert(0, this.createColorpicker(colorpickerWidth));
-        combo.width = combo.width + colorpickerWidth
+        if (this.useColorPicker){
```
Comment (ger-benjamin) [inline core/src/script/CGXP/plugins/FullTextSearch.js]: Add a space before the bracket "{"

File: core/src/script/CGXP/plugins/FullTextSearch.js [hunk 2]
```diff
@@ -178,9 +186,11 @@ cgxp.plugins.FullTextSearch = Ext.extend(gxp.plugins.Tool, {
...
+            colorpickerWidth = 85
```
Comment (ger-benjamin) [inline core/src/script/CGXP/plugins/FullTextSearch.js]: End line with a semicolon here...

File: core/src/script/CGXP/plugins/FullTextSearch.js [hunk 3]
```diff
@@ -178,9 +186,11 @@ cgxp.plugins.FullTextSearch = Ext.extend(gxp.plugins.Tool, {
...
+            combo.items.add(this.createColorpicker(colorpickerWidth));
+            combo.width = combo.width + colorpickerWidth
```
Comment (ger-benjamin) [inline core/src/script/CGXP/plugins/FullTextSearch.js]: ... and here :-)

**Comment by ger-benjamin** [comment]
I've added some comments.

Please merge after that. Thanks.

**Pull request merged**